source/CardGames.py

```diff
@@ -102,7 +102,13 @@ def addCard(self, card: Card, isKnown: bool = True):
 
   def clearHand(self):
     self.hand = []
-    self.knownCards = []
+    self.knownCards = [1]
+
```

Comment:
The function looks great! Clean and concise.

I think we are all in agreement that we are going to use comments next sprint as well.